test-framework/src/test/java/org/apache/brooklyn/test/framework/TestSshCommandTest.java

```diff
@@ -278,6 +274,28 @@ public void shouldFailIfTestEntityHasNoMachine() throws Exception {
     }
     
     @Test
+    public void shouldFailFastIfNoCommand() throws Exception {
+        Map<String, ?> equalsZero = ImmutableMap.of(EQUALS, 0);
+        
+        TestSshCommand test = app.createAndManageChild(EntitySpec.create(TestSshCommand.class)
+                .configure(TIMEOUT, Duration.ONE_MINUTE)
```

Comment:
The test passes for 0.5 s on my side so I think it is reasonable to shorten the limit to ten seconds?